index.html

```diff
@@ -147,7 +142,7 @@
                 id="q17"
                 data-name="Mobile Portfolio"
                 class="q17"
-              ><div key="0">Mobile Portfolio</div>
+              ><div>Mobile Portfolio</div>
               </div>
               </div>
               <div class="outer_q18">
```

Comment:
- [ ] [OPTIONAL] Every project title should be a heading tag. Consider `<h4>` to `<h6>`. Let's refactor this.